tirocks/src/env.rs

```diff
@@ -0,0 +1,201 @@
+// Copyright 2022 TiKV Project Authors. Licensed under Apache-2.0.
+
+mod inspected;
+pub mod logger;
+mod sequential_file;
+
+use self::inspected::DBFileSystemInspector;
+use crate::{Code, Result, Status};
+use libc::c_char;
+use tirocks_sys::{ffi_try, r};
+
+pub use self::inspected::FileSystemInspector;
+pub use self::sequential_file::SequentialFile;
+
+pub type IoPriority = tirocks_sys::rocksdb_Env_IOPriority;
+pub type Priority = tirocks_sys::rocksdb_Env_Priority;
+
+/// Options while opening a file to read/write
+// TODO: perhaps use the C struct directly
+pub struct EnvOptions {
```

Comment:
I'm a little confused about the naming convention. CamelCase used to mean native types (now extends to transparent wrapper at rust side), shouldn't we use snake_case here to represent a pointer wrapper?

Reply:
I don't get it. 

> CamelCase used to mean native types 

I don't think there is such convention.

> shouldn't we use snake_case here to represent a pointer wrapper?

In Rust, the style is always use CammelCase to name a struct, no matter what it represents.

Reply:
Ok, it's just my personal impression then.